crates/core/src/surfnet/locker.rs

```diff
@@ -1605,6 +1620,42 @@ impl SurfnetSvmLocker {
             }
         });
     }
+
+    /// Resets an account in the SVM state
+    ///
+    /// This function coordinates the reset of accounts by calling the SVM's reset_account method.
+    /// It handles program accounts (including their program data accounts) and can optionally
+    /// cascade the reset to all accounts owned by a program.
+    pub fn reset_account(&self, pubkey: Pubkey, config: ResetAccountConfig) -> SurfpoolResult<()> {
+        let cascade_to_owned = config.cascade_to_owned.unwrap_or_else(|| false);
```

Comment:
The default value for a bool is false, so let's do `unwrap_or_default()` here instead. I know it amounts to the same thing, but it implies "the struct handles the default case"